crates/nargo/src/cli/verify_cmd.rs

```diff
@@ -1,12 +1,12 @@
-use super::{compile_cmd::compile_circuit, read_inputs_from_file};
+use super::{compile_cmd::compile_circuit, prove_cmd::AbiMap, read_inputs_from_file};
```

Comment:
This makes me feel like those types should be bumped to `mod.rs`. They're more generic rather than being tied to proving in particular.